Tests/StreamChatTests/Workers/CurrentUserUpdater_Tests.swift

```diff
@@ -643,4 +643,51 @@ final class CurrentUserUpdater_Tests: XCTestCase {
         // THEN
         AssertAsync.willBeEqual(completionCalledError as? TestError, error)
     }
+    
+    // MARK: - Delete Local Downloads
+    
+    func test_deleteAllLocalAttachmentDownloads_success() throws {
+        let storedFileCount: () -> Int = {
+            let paths = try? FileManager.default.subpathsOfDirectory(atPath: URL.streamAttachmentDownloadsDirectory.path)
+            return paths?.count ?? 0
+        }
+        try FileManager.default.removeItem(at: .streamAttachmentDownloadsDirectory)
```

Comment:
hm, this test fails

Reply:
Need to review this one and update snapshots because I changed alignment in the item view.